### PR TITLE
Model: Implemented FileOutputTask

### LINKED ISSUE #196: Missing FileOutputTask and DataPacket classes
**Actual Behaviour**
Missing FileOutputTask and DataPacket classes

**Expected Behaviour**
These classes need to be ported and implemented as a part of the model subpackage.

**Would you like to work on the issue?**
Yes, I would work with this issue.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #196 

**Changes**: Implementing the FileOutputTask in the model subpackage.

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by jddeep** [comment]
@CloudyPadmal @mariobehling Please review.

**Comment by mariobehling** [comment]
Don't ping people extensively! Get co-developers in the community to review PRs.

File: app/src/main/java/io/neurolab/model/FileOutputTask.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    for (int c = 0; c < currentPacket.length; c++)
+                        out.print(currentPacket[c] + ",");
+                    out.println("");
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
```
Comment (suresh-gandhi) [inline app/src/main/java/io/neurolab/model/FileOutputTask.java:47]: @jddeep Remove an extra line of space here.

**Review by suresh-gandhi — CHANGES_REQUESTED** [review]

**Review by CloudyPadmal — APPROVED** [review]

File: app/src/main/java/io/neurolab/model/FileOutputTask.java (hunk 1 above)
Comment (jddeep) [reply app/src/main/java/io/neurolab/model/FileOutputTask.java:47]: Thanks @suresh-gandhi . Done

Comment (jddeep) [reply app/src/main/java/io/neurolab/model/FileOutputTask.java:47]: Yeah @suresh-gandhi done! Thanks.

**Comment by jddeep** [comment]
@CloudyPadmal This can be merged now.

**Pull request merged**